src/daemon/db_util.rs

```diff
@@ -214,6 +219,110 @@ pub async fn download_to(url: &Url, destination: &Path) -> anyhow::Result<()> {
     Ok(())
 }
 
+#[derive(Debug, Clone)]
```

Comment:
I'd move those utils outside of this module. They don't seem to fit in `db_util`.

Reply:
Moved to under `crate::utils::net`. Please re-approve if it looks good.